src/moment_kinetics_input.jl

```diff
@@ -197,27 +92,26 @@ function mk_input(scan_input=Dict(); save_inputs_to_txt=false, ignore_MPI=true)
     composition.Er_constant = get(scan_input, "Er_constant", 0.0)
 
     # Get reference parameters for normalizations
-    reference_parameter_section = set_defaults_and_check_section!(
+    reference_parameter_section = copy(set_defaults_and_check_section!(
```

Comment:
This is probably the wrong place to discuss this, but I don't understand why we need reference parameters in the code itself.

Reply:
My reasoning is that we need the reference parameters to calculate the collision frequency, so for me it's easier to understand if it's explicit what the reference parameters are for a given simulation (i.e. given in the input file or set by defaults), and 'everything else' is forced to be consistent with them. At the moment it's 'just' (I guess?) rhostar and collision frequency at reference parameters that are actually being used, but at some point we might need/want for example atomic rates (ionization, charge exchange, etc.) that are tabulated functions of density and temperature.

Another motivation is that we might want a post processing routine that calculates things in SI units, like the heat flux to the target (in MW/m^2), or the power input into the simulation (in MW). Then we'd need to know what the reference parameters are, so it's simplest if they are explicitly specified.

What's the downside of having reference parameters specified?